mantis-runtime/src/main/java/io/mantisrx/runtime/Context.java

```diff
@@ -159,5 +167,5 @@ public Observable<WorkerMap> getWorkerMapObservable() {
         return this.workerMapObservable;
     }
 
-
+    public ClassLoader getClassLoader() { return this.classLoader; }
```

Comment:
Is the idea to have this "custom" cl as nullable? It might be a bit confusing for users to use it without knowing whether or not it's appropriate.